### PR TITLE
PEP8 in sims #884 

### PR DESCRIPTION
**Pull request by theaverageguy**
Fixes https://github.com/nipy/dipy/issues/884

### CODE REVIEW

File: dipy/sims/tests/test_phantom.py [hunk 1]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
     FA[np.isnan(FA)] = 0
     # 686 -> expected FA given diffusivities of [1500, 400, 400]
     l1, l2, l3 = 1500e-6, 400e-6, 400e-6
-    expected_fa =  (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + (l2-l3)**2 + (l3-l1)**2 )/np.sqrt(l1**2 + l2**2 + l3**2))
+    expected_fa = np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + (l2 - l3)**2 + (l3 - l1)**2) / np.sqrt(l1**2 + l2**2 + l3**2))
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: This line is probably still >80 characters long. A suggestion: 

```
    expected_fa = (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + 
                   (l2 - l3)**2 + (l3 - l1)**2) / 
                   np.sqrt(l1**2 + l2**2 + l3**2)))

````

File: dipy/sims/tests/test_phantom.py [hunk 2]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
...
 
-    assert_array_almost_equal(FA.max(), expected_fa, decimal=2)
+    assert_array_almost_equal(FA.max(), expected_fa, decimal = 2)
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: No space needed here.

File: dipy/sims/tests/test_phantom.py [hunk 3]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
...
 
 
 def test_add_noise():
     np.random.seed(1980)
 
-    N = 50
-    S0 = 100
+    N=50
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: But space would be nice here.

File: dipy/sims/tests/test_phantom.py [hunk 4]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
...
-                   radii=np.linspace(0.2, 2, 6),
-                   S0=S0)
+    options=dict(func = f,
+                   t = np.linspace(0, 2 * np.pi, N),
+                   datashape = (10, 10, 10, len(bvals)),
+                   origin = (5, 5, 5),
+                   scale = (3, 3, 3),
+                   angles = np.linspace(0, 2 * np.pi, 16),
+                   radii = np.linspace(0.2, 2, 6),
+                   S0 = S0)
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: No spaces around "=" sign when assigning to a dict or when passing arguments to a function, please.

File: dipy/sims/tests/test_phantom.py [hunk 5]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
...
 
-    vol = orbital_phantom(gtab, **options)
+    vol=orbital_phantom(gtab, **options)
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: But space around "=" when used in assignment!

File: dipy/sims/tests/test_phantom.py [hunk 6]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
...
 
     for snr in [10, 20, 30, 50]:
-        vol_noise = orbital_phantom(gtab, snr=snr, **options)
+        vol_noise=orbital_phantom(gtab, snr = snr, **options)
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: Same as above.

File: dipy/sims/tests/test_phantom.py [hunk 7]
```diff
@@ -55,32 +55,32 @@ def test_phantom():
...
 
-        sigma = S0 / snr
+        sigma=S0 / snr
```
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: And same here.

**Comment by arokem** [comment]
Thanks for taking a look! Just a couple of small comments.

**Comment by theaverageguy** [comment]
Fixing em in 5! :)

**Comment by theaverageguy** [comment]
Can you check now @arokem ?

File: dipy/sims/phantom.py [hunk 8]
```diff
@@ -197,54 +197,54 @@ def orbital_phantom(gtab=None,
...
+    # """
 
-    #r=fvtk.ren()
-    #fvtk.add(r,fvtk.volume(vol234[...,0]))
-    #fvtk.show(r)
-    #vol234n=add_rician_noise(vol234,20)
+    # r=fvtk.ren()
+    # fvtk.add(r,fvtk.volume(vol234[...,0]))
+    # fvtk.show(r)
+    # vol234n=add_rician_noise(vol234,20)
```
Comment (arokem) [inline dipy/sims/phantom.py]: Somebody might want to clean this up at some point, but maybe not on this PR.

**Comment by arokem** [comment]
LGTM. Someone else want to take a look here?

File: dipy/sims/tests/test_phantom.py [hunk 9]
```diff
@@ -55,7 +55,9 @@ def test_phantom():
     FA[np.isnan(FA)] = 0
     # 686 -> expected FA given diffusivities of [1500, 400, 400]
     l1, l2, l3 = 1500e-6, 400e-6, 400e-6
-    expected_fa =  (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + (l2-l3)**2 + (l3-l1)**2 )/np.sqrt(l1**2 + l2**2 + l3**2))
+    expected_fa = (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + 
+                   (l2 - l3)**2 + (l3 - l1)**2) / 
+                   np.sqrt(l1**2 + l2**2 + l3**2)))
```
Comment (manu-tej) [inline dipy/sims/tests/test_phantom.py]: `expected_fa = (np.sqrt(0.5) * np.sqrt((l1 - l2)**2 + 
 +                   (l2 - l3)**2 + (l3 - l1)**2) / 
 +                   np.sqrt(l1**2 + l2**2 + l3**2)))`

Comment (manu-tej) [inline dipy/sims/tests/test_phantom.py]: remove extra brackets.

File: dipy/sims/voxel.py [hunk 10]
```diff
@@ -298,16 +298,15 @@ def multi_tensor(gtab, mevals, S0=100, angles=[(0, 0), (90, 0)],
     sticks = _check_directions(angles)
 
     for i in range(len(fractions)):
-            S = S + fractions[i] * single_tensor(gtab, S0=S0, evals=mevals[i],
-                                                 evecs=all_tensor_evecs(
-                                                     sticks[i]), snr=None)
+        S = S + fractions[i] * single_tensor(gtab, S0=S0, evals=mevals[i],
+                                             evecs=all_tensor_evecs(
```
Comment (samuelstjean) [inline dipy/sims/voxel.py]: put the function and its param on the same line, it's a bit hard to read like that

File: dipy/sims/tests/test_phantom.py (hunk 9 above)
Comment (arokem) [inline dipy/sims/tests/test_phantom.py]: Yup. That's probably my bad for suggesting this. See failing tests: 

https://travis-ci.org/nipy/dipy/jobs/113234014#L2306

**Comment by arokem** [comment]
Closing for now.

**Pull request closed without merging**